datalad/tests/test_cmdline_main.py

```diff
@@ -66,8 +66,7 @@ def test_version():
 
 
 def test_help_np():
-    with patch.dict('os.environ', {'DATALAD_HELP2MAN': '1'}):
```

Comment:
I thought that there was some specifixg handling based on this variable... But if test passes I guess all is OK